src/pages/oversikt/kandidatliste/IngenKandidater.less

```diff
@@ -0,0 +1,9 @@
+.IngenKandidater {
```

Comment:
Vi bruker BEM-standarden når vi lager klassenavn. Vi har en bemHelper som lager de klassenavnene for oss. Se f.eks. i Kandidatliste, der står det "const cls = bemHelper('kandidatliste');".